project/invitro/exports.py

```diff
@@ -1,9 +1,28 @@
+from copy import copy
 from django.db.models.loading import get_model
 
 from utils.helper import FlatFileExporter
 
 
-class IVEndpointFlatDataPivot(FlatFileExporter):
```

Comment:
invitro.views - line 42 calls IVEndpointFlatDataPivot, should that be changed to DataPivotEndpoint?